FreeRTOS/iot-reference-stm32u5
# Fix HardFault when no root_ca_cert is present

Description
-----------
Address the HardFault case reported in #84.

Test Steps
-----------
To reproduce the original issue, manually provision a device using the steps in the Getting Started Guide, but do not provision a Root CA Certificate. Note that the device HardFaults.

Checklist:
----------

Related Issue
-----------
Issue #84 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
